HandheldCompanion/Controls/GameView.xaml.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace HandheldCompanion.Controls
+{
+    /// <summary>
+    /// Interaction logic for GameView.xaml
+    /// </summary>
+    public partial class GameView : UserControl
+    {
+        public GameView()
+        {
+            InitializeComponent();
```

Comment:
Where are you setting your DataContext?

Reply:
Library page

Reply:
@Xirdneth Not a fan of it. Also GameView shouldn't really exist as it's a singular child on the library page.
Should probably be put into LibraryPage.